src/python/pants/backend/python/rules/BUILD

```diff
@@ -30,12 +31,14 @@ python_tests(
     'src/python/pants/engine/legacy:graph',
     'src/python/pants/engine/legacy:structs',
     'src/python/pants/rules/core',
+    'src/python/pants/source',
     'src/python/pants/testutil:test_base',
     'src/python/pants/testutil/engine:util',
     'src/python/pants/testutil/subsystem',
     'src/python/pants/util:collections',
     'src/python/pants/util:contextutil',
   ],
+  timeout = 120,
```

Comment:
Is this just for testing, or does the target take twice as long to run now?

Reply:
The unit test takes about 55 seconds to run locally (whereas the IT takes about 90). This target applies to all unit tests for the file and takes 90 seconds locally. I put it at 120 to reduce the risk of flakes, e.g. if RBE is slower than us.

Reply:
Thanks!! This seems appropriate -- I think john's comment below would be good to think about too though as it's also relevant to runtime!